app/assets/src/components/utils/Metrics.js

```diff
@@ -0,0 +1,54 @@
+const metric_names = {
+  "NT.aggregatescore": "Score",
+  "NT.rpm": "NT rPM",
+  "NT.r": "NT r (reads)",
+  "NT.zscore": "NT z-score",
+  "NT.maxzscore": "Max z-score",
+  "NT.percentidentity": "NT %id",
+  "NT.neglogevalue": "NT log(1/e)",
+  "NR.rpm": "NR rPM",
+  "NR.r": "NR r (reads)",
+  "NR.zscore": "NR z-score",
+  "NR.percentidentity": "NR %id",
+  "NR.neglogevalue": "NR log(1/e)"
+};
+const metric_names_augmented = {};
+for (var m in metric_names) {
+  metric_names_augmented[m] = metric_names[m];
+  metric_names_augmented[m.replace(".", "_")] = metric_names[m];
+  metric_names_augmented[m.replace(".", "_").toLowerCase()] = metric_names[m];
+}
+metric_names_augmented["aggregatescore"] =
+  metric_names_augmented["NT.aggregatescore"];
+export const METRIC_NAMES = metric_names_augmented;
+
+export const THRESHOLD_METRICS = [
```

Comment:
These are the name that will show up? 
Should we use proper spacing and words, i.e. for instance `NT Percent Identity`